core/dbt/parser/schemas.py

```diff
@@ -886,11 +887,13 @@ def parse_patch(
                     # re-application of the patch in partial parsing.
                     node.patch_path = source_file.file_id
             else:
-                raise ParsingException(
+                msg = (
                     f"Did not find matching node for patch with name '{patch.name}' "
                     f"in the '{patch.yaml_key}' section of "
                     f"file '{source_file.path.original_file_path}'"
                 )
+                logger.info(msg)
```

Comment:
As a compromise, what do you think about making this:
```python
warn_or_error(msg, log_fmt=warning_tag('{}'))
```

That's what we do in other places when people have unused resources, or tests that depend on a disabled node

Reply:
Ok